plugins/wordpress-rule-exclusions-before.conf

```diff
@@ -1120,4 +1120,34 @@ SecRule REQUEST_FILENAME "@endsWith /wp-admin/admin-ajax.php" \
             "t:none,\
             ctl:ruleRemoveTargetById=942360;ARGS:s"
 
+# Uploading plugins as a zip file
+SecRule REQUEST_FILENAME "@endsWith /wp-admin/update.php" \
+    "id:9507973,\
+    phase:2,\
+    pass,\
+    t:none,\
+    nolog,\
+    ver:'wordpress-rule-exclusions-plugin/1.0.1',\
+    chain"
+    SecRule ARGS:action "@streq upload-plugin" \
+        "t:none,\
+        chain"
+        SecRule FILES:pluginzip "@endsWith .zip" \
```

Comment:
`FILES` is not considered a collection by the parser. @airween is that a mistake? The documentation isn't clear on that, but I think that all `FILES...` variables should be considered collections by the parser.

Reply:
Ping @airween.

Reply:
FYI, a fix for seclan_parser has been merged. We just need to wait for a new release.

Reply:
It's done.

https://github.com/coreruleset/secrules_parsing/releases/tag/v0.2.11